source/brailleDisplayDrivers/hidBrailleStandard.py

```diff
@@ -103,16 +104,27 @@ def __init__(self, port="auto"):
 				continue  # Couldn't connect.
 			if self._dev.usagePage != HID_USAGE_PAGE_BRAILLE:
 				log.debug("Not braille")
+				self._dev.close()
 				continue
 			cellValueCaps = self._findCellValueCaps()
-			if cellValueCaps:
+			if len(cellValueCaps) > 0:
+				if any(x.ReportCount != cellValueCaps[0].ReportCount for x in cellValueCaps):
+					log.warn("Found multi-line display with an irregular shape, ignoring.")
```

Comment:
[`log.warn` is deprecated.](https://docs.python.org/3/library/logging.html#logging.Logger.warning)

```suggestion
					log.warning("Found multi-line display with an irregular shape, ignoring.")
```